### ISSUE TITLE
Documentation problem: just what is a full scan?

### ISSUE THREAD

**Issue by LinuxOnTheDesktop** [Nefelim4ag/Ananicy#277]
Thank you for your nice nicer.

Unless I miss something - and admittedly reading the code would tell me this - the meaning of the conf file's 'run full system scan' is not explained. That phrase seems to imply that Ananicy will do something - if necessary - even between such 'full scans'. But what?

I was prompted to investigate this matter by an old laptop of mine using 2% of its CPU every Ananicy scan. So I asked myself: why not make the scan less frequent? And answer there was none.

**Comment by aviallon** [comment]
A full scan is basically Ananicy processing every running process, comparing its command to the rules currently loaded, and performing the appropriate actions on it. This is quite costly in CPU time if there is a lot if processes running.

**Comment by LinuxOnTheDesktop** [comment]
Right. I think it would be helpful were the ananicy.conf file and/or the README to make that point.
